### ISSUE TITLE
Loqus DB links to homozygous cases

### ISSUE THREAD

**Issue by Jakob37** [Clinical-Genomics/scout#4331]
A request from our geneticists.

At the moment, the number of observations display 

| ID  | Nr obs. | Nr homo. | Total nr. |
| --- | ------- | -------- | --------- |
| ID  | 3       | 2        | 5000      |

Below is a line as such for the three heterozygous observations:

```
Case 1, case 2, case 3
```

But the homozygous cases are not shown. Could that be displayed as well in a similar way?

**Comment by dnil** [comment]
It would be nice. loqusdb does not yet support logging which samples were homozygous, only which samples had the variant, and for recessive variants that are somewhat common in the population it quickly gets old clicking through all the cases. That would be this issue: https://github.com/Clinical-Genomics/loqusdb/issues/115. I'll note the support, and perhaps keep this one open as well so it doesn't get lost - we will need changes to loqusdbapi and scout as well to make it work. As usual, unclaimed issues are very much up for grabs. 😁

**Comment by Jakob37** [comment]
OK I see! I'll forward that info!

**Comment by dnil** [comment]
I just had a clinician ask the same question, for a late onset recessive disorder. We should fix it in loqus, but for all the retroactive we could also add a quick (sic) search in scout. It would really only need to find the variants with the same variant_id (indexed) that have an `AR_hom` inheritance model. This is however also connected to deciding on a format for displaying cohorts or variants from multiple cases. We could reuse the SNV&SV page, add a search by variant id and filter by inheritance model, or bump a slightly different one intended for cohort research.

**Comment by TJO-Lund** [comment]
I am looking at a variant in i Scout with 134 Local observations of which 1 is homozygous. In Scout I see about 50 samples (not all 134) and I have no way in Scout to find out which sample-ID is the homozygous one. I am writing this just to lift the issue again (as the last comment by Daniel was quite a while ago in 2025). Can this issue perhaps get some priority :-) as it is very frustrating (at least to me) and I also wish not to have to disturb our bioinformaticians to find the homozygous sample.

**Comment by dnil** [comment]
Yes, it would be nice - see https://github.com/Clinical-Genomics/loqusdb/issues/115. Maybe 2026 is the year for it? Happy new year!
